.circleci/push.bash.j2

```diff
@@ -5,27 +5,27 @@ export NEW_POINT_RELEASE=true
 
 IFS="," read -ra DOCKER_TAGS \<<< "<< parameters.extra_tags >>"
 
-echo "DOCKER_TAGS: ${DOCKER_TAGS[@]}"
+echo "DOCKER_TAGS: " "${DOCKER_TAGS[@]}"
```

Comment:
Nit:

```suggestion
echo "DOCKER_TAGS: ${DOCKER_TAGS[@]}"
```

Reply:
Shellcheck has [an opinion on concatenating strings and arrays](https://www.shellcheck.net/wiki/SC2145), but I'll change it back. :+1: